packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -281,13 +284,14 @@ export class IndexedDbPersistence implements Persistence {
       this.indexManager,
       /*keepDocumentChangeLog=*/ this.allowTabSynchronization
     );
+    this.window = platform.window;
     if (platform.window && platform.window.localStorage) {
-      this.window = platform.window;
-      this.webStorage = this.window.localStorage;
+      this.webStorage = platform.window.localStorage;
     } else {
-      throw new FirestoreError(
-        Code.UNIMPLEMENTED,
-        'IndexedDB persistence is only available on platforms that support LocalStorage.'
+      this.webStorage = null;
+      log.error(
+        LOG_TAG,
+        'LocalStorage is unavailable. Multi-tab functionality may not be supported.'
```

Comment:
They have not enabled multi-tab (since we would have thrown a different error already if they did).  So I think this error should say, 'LocalStorage is unavailable. As a result, persistence may not work reliably. In particular enablePersistence() could fail immediately after refreshing the page.'

or something like that (maybe we should tell them they can enable `force`?  Not sure...  Probably best to leave it ambiguous and then if we get feedback, find out how people are hitting this and what advice we should be giving them.)